addon/classify-macro.js

```diff
@@ -0,0 +1,10 @@
+import computed from 'ember-computed';
+import get from 'ember-metal/get';
+
+export default function classifyMacro(key, ifFalsey, ifTruthy) {
```

Comment:
There are no tests for this yet...to be added. But, this is a very simple computed property so that we can safely replace `classNameBindings: ['isOpen:@thing1:@thing2']` with the equivalent made up of many class names.